2-Services/2.3-ensure-nonessential-services-are-removed-or-masked.bats

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bats
+
+@test "2.3 Ensure nonessential services are removed or masked (Manual)" {
+    skip "This audit has to be done manually"
+
+    # Run the following command and review ports/network-configuration:
+    # lsof -i -P -n | grep -v "(ESTABLISHED)"
```

Comment:
Maybe we can just use the advise text from the CIS document
```suggestion
    # lsof -i -P -n | grep -v "(ESTABLISHED)"
    # Review the output to ensure that all services listed are required on the system. If a listed
    # service is not required, remove the package containing the service. If the package
    # containing a non-essential service is required, stop and mask the non-essential service.
```

Reply:
I do not have any preferences, I will implement these changes